web-ui/src/components/profile/Profile.jsx

```diff
@@ -112,6 +112,60 @@ const Profile = ({ memberId, pdlId, checkinPdlId }) => {
     }
   }, [csrf, supervisorid]);
 
+  const handleSaveNewOrganization = async () => {
+    const res = await resolve({
+      method: 'POST',
+      url: '/services/volunteer/organization',
+      headers: {
+        'X-CSRF-Header': csrf,
+        'Content-Type': 'application/json'
+      },
+      data: newOrganization
+    });
```

Comment:
Code like this we generally pull up into a file in the `web-ui/src/api` directory. I would put this and the one below for events in a file together called `volunteer.js`, if it were me.